packages/angular/src/components/alert.component.ts

```diff
@@ -6,14 +6,16 @@
 import {
   Component,
   ElementRef,
-  EventEmitter,
-  Input,
   NgZone,
+  Input,
   Output,
+  EventEmitter,
 } from '@angular/core';
-import type {
-  SynAfterHideEvent, SynAfterShowEvent, SynAlert, SynHideEvent, SynShowEvent,
-} from '@synergy-design-system/components';
+import type { SynAlert } from '@synergy-design-system/components';
+import type { SynShowEvent } from '@synergy-design-system/components';
+import type { SynAfterShowEvent } from '@synergy-design-system/components';
+import type { SynHideEvent } from '@synergy-design-system/components';
+import type { SynAfterHideEvent } from '@synergy-design-system/components';
```

Comment:
Could we get a problem with importing everything for its own instead of doing this: 

```js
import type {
  SynAfterHideEvent, SynAfterShowEvent, SynAlert, SynHideEvent, SynShowEvent,
} from '@synergy-design-system/components';
```

If don't think so, right?

Reply:
No, there should not be a problem at all with this, even if I am unsure why it does that. However, as can be seen in [the react wrappers](https://github.com/synergy-design-system/synergy-design-system/blob/main/packages/react/src/components/alert.ts), they always behaved that way.

While having a look at #553, I also thought this would crash. However, the types will always be stripped via typescript, so it will not effect the bundle size in either way.